src/aiida/cmdline/commands/cmd_process.py

```diff
@@ -525,7 +525,7 @@ def process_repair(manager, broker, dry_run):
     """
     from aiida.engine.processes.control import get_active_processes, get_process_tasks, iterate_process_tasks
 
-    active_processes = get_active_processes(project='id')
+    active_processes = [process.pk for process in get_active_processes() if process.user.is_default]
```

Comment:
```suggestion
    active_processes = (process.pk for process in get_active_processes() if process.user.is_default)
```

the process will be evaluated and the list can be large, use iterator instead.

Reply:
Sorry, haven't had a chance to get back to this. The thing is, somehow in the tests, process is evaluated as an int. Unclear why. Also, maybe best to modify the active process query to optionally filter by the default user, so this list comprehension filtering could be avoided all together. Will think about this later this week.